i18n/en.json

```diff
@@ -100,6 +100,7 @@
 	"scratch-confirmaccount-email-resent": "Confirmation email has been re-sent.",
 	"scratch-confirmaccount-new-scratcher": "New Scratchers cannot submit account requests.",
 	"scratch-confirmaccount-joinedat": "You must have been registered on Scratch for at least $1 days to request an account.",
+	"scratch-confirmaccount-profile-error": "There was an error while checking your Scratch account. Try again later.",
```

Comment:
Maybe this could suggest possible causes for the error? (e.g. "You may be banned")

Reply:
Banned users should not see the error, because this is checked after the comment. One possible scenario is that scratchr2 is down but API and scratch-www is not.

Reply:
I don't think this is a huge issue, it's just a general technical failure indicator.